packages/wonder-blocks-form/src/components/__docs__/labeled-text-field.stories.js

```diff
@@ -528,6 +528,27 @@ CustomStyle.parameters = {
     },
 };
 
+export const WithMarkup: StoryComponentType = (args) => {
+    return (
+        <LabeledTextField
+            {...args}
+            label="Name"
+            description={
+                <span>
+                    Please enter your <strong>name</strong>
+                </span>
+            }
+        />
+    );
+};
+
+WithMarkup.parameters = {
+    docs: {
+        storyDescription: `\`LabeledTextField\`'s \`label\` and \`description\` props
+        can accept \`React.Node\`s.`,
```

Comment:
**suggestion:** It'd be helpful adding a bit more context.... something like: .... `This is helpful when you need to decorate or use specific elements in your form field (e.g. including Popovers, Tooltips or emphasized text)`.